app/main.js

```diff
@@ -67,9 +71,9 @@ function createTrayIcon () {
   } else {
     appIcon = new Tray(iconFolder + '/stretchly_18x18.png')
   }
-  appIcon.setToolTip('stretchly - break time reminder app')
   isPaused = false
   appIcon.setContextMenu(getTrayMenu())
+  toolTipUpdater = setInterval(updateToolTip, 1000)
```

Comment:
Contemplating whether, since the time shown is already a rough estimate, the toolTipUpdater should also run at a longer interval, maybe 5 or 10 seconds instead of just one.